pyuvdata/uvdata/ms.py

```diff
@@ -144,6 +1608,7 @@ def read_ms(
         strict_uvw_antpos_check=False,
         use_old_phase=False,
         fix_phase=False,
+        ignore_single_chan=True,
```

Comment:
These two parameters are not documented in the docstring.

Reply:
Documentation added in 3068e067.